aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst

```diff
@@ -148,96 +188,53 @@ To do so, start by creating a ``cluster.yaml`` with the following content
     name: procluster
     region: us-east-1
     version: 'YOUR_EKS_VERSION'
-    managedNodeGroups:
-    - name: ng-procluster
-    instanceType: t3.small
-    desiredCapacity: 2
-    labels: {role: worker}
-    ami: ami-xxxxx
-    amiFamily: AmazonLinux2
-    ssh:
-        publicKeyName: yoursshkeyname
-    overrideBootstrapCommand: |
-        #!/bin/bash
-        sudo /etc/eks/bootstrap.sh procluster
 
-This config file allows you to create a cluster using the AMI from the previous step,
-with two nodes and SSH access.
 
-Also we use AmazonLinux2 in amiFamily because at this date it's the only native option support by eksctl.
+Add the following content to your file
 
-The "overrideBootstrapCommand" lets you launch the bootstrap script from AWS EKS
-to initialise the nodes. For further cluster customisation see `this <https://eksctl.io/>`_.
 
+.. tabs::
 
```

Comment:
Can we reorder the tabs so that they match the order in the remaining text? i.e. 'Without FIPS' first and then 'With FIPS'